operator/pkg/install/wait.go

```diff
@@ -53,27 +44,21 @@ type deployment struct {
 
 // WaitForResources polls to get the current status of all pods, PVCs, and Services
```

Comment:
We aren't explicitly waiting for PVCs here, AFAIK? That may have changed at some point.

In general though I would like to reuse helm wait logic 100% here eventually as well, these are the kinds of things where we have no real need to be special.

Reply:
I was very close to swapping this out with the Helm wait logic actually. I think there were some awkward API differences when I first started the refactoring, but those are likely resolved now (after more refactoring) and could be tackled now.

Though one thing we do that is a bit nice is give some info on _why_ its not ready (extractPodFailureReason), like I think we can pull up the "image pull backoff" for instance. Probably the best approach would be to upstream that.

Reply:
Updated the comment for now at least.